04_model_inference.ipynb

```diff
@@ -395,25 +395,36 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": 6,
+   "id": "7c1ec213",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels = cc.read_from_ceph(MODEL_KEY, \"label-map\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
    "id": "49821583",
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Our Model Predicts this PR to be in category [0]\n"
+      "Our model predicts this PR could be merge in this time frame ['0:00:00 - 0:02:00']\n"
      ]
     }
    ],
    "source": [
     "prediction = model.predict(pr_df)\n",
+    "label = labels[labels[\"Class_Label\"] == prediction[0]][\"Class_Name\"].values\n",
     "if ACTION:\n",
     "    pr = repo.get_pull(pr_ids[0].number)\n",
-    "    pr.create_issue_comment(f\"Our Model Predicts this PR to be in category {prediction}\")\n",
+    "    pr.create_issue_comment(f\"Our model predicts this PR could be merge in this time frame{label}\")\n",
     "else:\n",
-    "    print(f\"Our Model Predicts this PR to be in category {prediction}\")\n"
+    "    print(f\"Our model predicts this PR could be merge in this time frame {label}\")\n"
```

Comment:
should this be 
```suggestion
    "    print(f\"Our model predicts this PR could be merged in the time frame {label}\")\n"
```
or "This PR is predicted to merge in the time frame {label} by our model"